Library/ViewModels/FindFriendsFriendFollowCellViewModelTests.swift

```diff
@@ -23,6 +23,9 @@ final class FindFriendsFriendFollowCellViewModelTests: TestCase {
   let hideFollowButton = TestObserver<Bool, NoError>()
   let hideUnfollowButton = TestObserver<Bool, NoError>()
   let hideProjectsCreated = TestObserver<Bool, NoError>()
+  let followButtonAccessibilityLabel = TestObserver<String, NoError>()
+  let unfollowButtonAccessibilityLabel = TestObserver<String, NoError>()
+  let cellAccessibilityLabel = TestObserver<String, NoError>()
```

Comment:
Generally we keep these alphabetical too as far as possible, makes it easier to find things later.